JS/Lesson26/onur/createNewPost/styles.css

```diff
@@ -0,0 +1,70 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    background-color: #dde7df;
+}
+
+.container {
+    background-color: rgb(199, 194, 194);
+    max-width: 800px;
+    margin: 5rem auto;
+    padding: 20px;
+    border-radius: 10px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+.form-container {
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    margin: 1rem 0;
+    font-size: 1.2rem;
+}
+
+.text {
+    padding: 0.5rem;
+    width: 500px;
+    border-radius: 4px;
+    font-size: 1rem;
+    border: 1px solid rgb(144, 185, 204);
+}
+
+.text:focus {
+    outline: 0;
+    border: 1px solid rgb(18, 215, 241);
+    box-shadow: 0 0 1px 1px #279fbd
+}
+
+h1 {
+    text-align: center;
+    color: #333;
```

Comment:
Suggestion for the future: in the beginning of the project, create variables for the colors that you will be using in the mentioned above main.css file and refer to variables everywhere where you need to use colors. It will make your code a lot more scalable.